openwisp_radius/receivers.py

```diff
@@ -105,7 +105,6 @@ def close_previous_radius_accounting_sessions(instance, created, **kwargs):
         unique_id=instance.unique_id
     ).filter(
         stop_time__isnull=True,
-        called_station_id=instance.called_station_id,
         calling_station_id=instance.calling_station_id,
         username=instance.username,
```

Comment:
Are we sure the `instance.username` we get here is the real username and not the mac address?
Please add some verbose logging in this part so we can debug this issue.